metastats-infrastructure/src/test/groovy/life.qbic.metastats/MetaStatsPresenterSpecification.groovy

```diff
@@ -0,0 +1,10 @@
+package life.qbic.metastats
+
+import spock.lang.Specification
+
+class MetaStatsPresenterSpecification extends Specification {
+
+    def "creates document content correctly"() {
+
```

Comment:
This test is currently not implemented, will this be extended in future?

Reply:
How to check for that? You would need a correct document for a fixed context and compare the output. This might be difficult.

Reply:
I realized that therefore I left the test empty and forgot to delete it. Done ✅